database/init_db.cypher

```diff
@@ -0,0 +1,42 @@
+//Delete all nodes and relationships
+MATCH (n)
+DETACH DELETE n
+
+//Create all nodes and relationships
+CREATE
+  (`Process 1`:Process {name:               'Kunde anlegen', description: 'Kunde anlegen im System',
+                        creation_timestamp: datetime(),
+                        last_timestamp:     datetime()})
+    -[:should {should: 10}]->(Ausfallzeit:Metric {name:        'Ausfallzeit',
+                                                  description: 'Durchschnittliche Ausfallzeit in Minuten'})
```

Comment:
Meiner Meinung nach fehlt da noch eine Angabe bei der Beschreibung der Ausfallzeit, nämlich ob die in Minuten pro Tag oder Minuten pro Jahr angegeben ist.

Vorschlag:
`                                                  description: 'Durchschnittliche Ausfallzeit in Minuten pro Tag'})
`